index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="styles.css">
+ <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital@1&display=swap" rel="stylesheet">
+<title>Portfolio</title>
+</head>
+<body>
+ <header> <nav id="menu-top">
+  <ul class="topnav">
+    <li class="topnav-logo"><a href="#">My Logo</a></li>
+    <li class="topnav-right"><a href="#">Home</a></li>
+    <li class="topnav-right"><a href="#">About</a></li>
+    <li class="topnav-right"><a href="#">Portfolio</a></li>
+    <li class="dropdownIcon"><a href="#">&#9776;</a></li>
+  </ul>
+</nav>  </header>
+ 
+ <div class="flex-container">
```

Comment:
- [ ] For this part, I invite you to use a `section` tag in place <div>. This could help the SEO part of the web page to improve the index and have a better position on searchers.